bin/addons/openupgrade_records/model/comparison_config.py

```diff
@@ -23,11 +23,25 @@
 #
 ##############################################################################
 
-from osv import osv, fields
 import openerplib
-from tools.translate import _
+import logging
 
-class openupgrade_comparison_config(osv.osv):
+try:
+    from openerp.addons.openupgrade_records.lib import apriori
+except ImportError:
+    from openupgrade_records.lib import apriori
+
+try:
+    from openerp.osv.orm import Model, except_orm
+    from openerp.osv import fields
+    from openerp.tools.translate import _
+except ImportError:
+    from osv.osv import osv as Model, except_osv as except_orm
```

Comment:
We know that this is going to fall always in this part for v5, but good strategy to keep all versions with the same code :wink: